src/middlewared/middlewared/plugins/reporting/events.py

```diff
@@ -64,9 +65,22 @@ def run_sync(self):
         while not self._cancel_sync.is_set():
             # this gathers the most recent metric recorded via netdata (for all charts)
             retries = 2
+            interfaces_names = [i['name'] for i in self.middleware.call_sync('interface.query')]
```

Comment:
This is too expensive to call every 0.5 seconds. That endpoint does a TON of other things that just doesn't make sense when all we're doing is getting the interface name.

We should extend `interface.query` with an `{'extra': {'names_only': True}}` argument that short-circuits very early and returns the names of the interfaces instead of doing everything else